servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -642,21 +618,30 @@ public FileBlockInfo getFileBlockInfo(long fileId, int fileBlockIndex)
     }
   }
 
+  /**
+   * Returns all the {@link FileBlockInfo} of the given file. Called via RPC, and internal masters.
```

Comment:
If the thrift server doesn't expose this but can only be called by the web ui, the comment should be updated.